0206-Reverse-Linked-List.py

```diff
@@ -0,0 +1,26 @@
+'''
+Reverse a singly linked list.
+
+Example:
+
+Input: 1->2->3->4->5->NULL
+Output: 5->4->3->2->1->NULL
+
+Follow up:
+
+A linked list can be reversed either iteratively or recursively. Could you implement both?
+'''
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, x):
+#         self.val = x
+#         self.next = None
+
+# Time Complexity - O(n), Space Complexity - O(1)
+class Solution:
+    def reverseList(self, head: ListNode) -> ListNode:
+        temp = None
```

Comment:
**temp**, as a name is very confusing, try to give some relatable name like in this case **previous**

Reply:
ok